### PR TITLE
Add field to posts, lastCkEditorUpdatedAt

### PR DESCRIPTION
**Pull request by lagerros**
Built after discussion with Oli. 

* Adds a field to posts, lastCkEditorUpdatedAt
* Run migration
* Adds an updateMutator to update this field upon receiving an event to the ckEditor webhook

Tested with a synthetic event and worked as expected. 

This will be used in another feature I'm working on.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1206201156644271) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/server/ckEditor/ckEditorWebhook.ts [hunk 1]
```diff
@@ -3,10 +3,62 @@ import { Posts } from '../../lib/collections/posts/collection';
...
 import { addStaticRoute } from '../vulcan-lib/staticRoutes';
 import { ckEditorApi, ckEditorApiHelpers, documentHelpers } from './ckEditorApi';
-import { createAdminContext, createMutator } from '../vulcan-lib';
+import { createAdminContext, createMutator, updateMutator } from '../vulcan-lib';
 import CkEditorUserSessions from '../../lib/collections/ckEditorUserSessions/collection';
 import { ckEditorUserSessionsEnabled } from '../../lib/betas';
 
+
+// Hacky lil' solution for now 
+const testWebHookLocally = async () => {
```
Comment (oetherington) [inline packages/lesswrong/server/ckEditor/ckEditorWebhook.ts:12]: It'd be cool if this developed into an integration test someday

File: packages/lesswrong/server/ckEditor/ckEditorWebhook.ts [hunk 2]
```diff
@@ -98,12 +150,25 @@ async function handleCkEditorWebhook(message: any) {
...
       await documentHelpers.saveOrUpdateDocumentRevision(postId, documentContents);
+      if (!!updated_at && !!postId) {
+        const adminContext = await createAdminContext()
+        await updateMutator({
+          collection: Posts,
+          documentId: postId,
+          set: {
+            lastCkEditorUpdatedAt: new Date(updated_at)
+          },
+          unset: {},
```
Comment (oetherington) [inline packages/lesswrong/server/ckEditor/ckEditorWebhook.ts:165]: Super minor nit: you can just exclude the `unset` if you're not using it.

**Review by oetherington — APPROVED** [review]
No objections here (on the assumption that the testing stuff will be cleaned up in a future PR).

In general it'd be nice to read the motivation behind a new field, but I guess that'll also be in the future PR.

**Comment by lagerros** [comment]
Yep. After discussing more with Robert and Oli I've run the down migration and will not be proceeding with the PR. 

I'd like to leave the testing stuff in there for now, because it's pretty useful for debugging the webhook locally (as there's no easier way to receiving a ckeditor webhook in the development environment). But agreed, would be nice to have it as an integration test!

**Pull request closed without merging**